collector/cluster_test.go

```diff
@@ -0,0 +1,913 @@
+package collector_test
```

Comment:
If you change this to `collector`, there's no need to change the visibility of some of the variables in `cluster.go` or `base.go` to be accessible from outside of the package.